src/test/TEST_ams/solvers.saved.lassen

```diff
@@ -1,117 +1,117 @@
 # Output file: solvers.out.0
```

Comment:
Big changes w.r.t to convergence. It is "expected" from the changes from G-S to Jacobi.